VEHICLE/SetVehicleExtra.md

```diff
@@ -10,14 +10,13 @@ void SET_VEHICLE_EXTRA(Vehicle vehicle, int extraId, BOOL disable);
 
 ```
 Note: only some vehicle have extras  
-extra ids are from 1 - 9 depending on the vehicle  
+extra ids are from 1 - 13 depending on the vehicle 
+-------------------------------------------------
+The method asks a question: Hide Extra?
+0 answers No 
```

Comment:
I don't want to sound pedantic, but it's currently declared as a Bool in the native method, so while it's true `1` and `0` represent bit on/off states, it should be `False` or `True` to conform with the native method declaration.

Also, if you're improving a native, it's better to just take the content out of the code fences and declare it as the description, since code fences are reserved for documentation coming from native imports and [native docs](https://docs.fivem.net/natives) will incorrectly mark it as "Has "imported" docs" instead of "Has "modern" docs" even though you made changes to it. 

There's more on the [contribution](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) guide.

Thanks for your contribution though! Maybe check on those tiny details I pointed out 🙂

For reference, here's the decompiled game native from 323

```c
int __fastcall sub_7FF7032FCCE0(signed int vehicle, int extraId, char disable)
{
  __int64 v5; // rax
  __int64 v6; // rbx

  v5 = getCVehicleFromScriptHandle(vehicle);
  v6 = v5;
  if ( v5 )
  {
    if ( (extraId - 1) <= 13 )
    {
      LODWORD(v5) = sub_7FF70355358C(v5, extraId + 290, disable);
      if ( disable )
      {
        if ( !*(v6 + 2680) )
        {
          LODWORD(v5) = sub_7FF70349A18C(v6, 0i64);
          if ( !v5 )
          {
            sub_7FF7034A4930(v6, 42i64);
            LODWORD(v5) = sub_7FF7034A4930(v6, 43i64);
          }
        }
      }
    }
  }
  return v5;
}
```